shell-completion/bash/systemd-nspawn

```diff
@@ -139,6 +140,10 @@ _systemd_nspawn() {
                                 _signals
                                 return
                         ;;
+                        --notify-ready)
+                                comps='true false'
```

Comment:
strictly speaking 0, false, no are all equivalent, and 1, true, yes too when systemd deals with booleans. You chose to complete things here with "true" and "false", for this bool. The other switches to nspawn usually used "yes" and "no" however. I think it's fine to stick to just one pair of positive and negative boolean values, instead of all 6 different values, but I think we really should try to use the same one for all completions, and as it looks right now that would be "yes" and "no", and not "true" and "false". Can you make thatquick change, please?